[DOCS] Renames limitation items in DFA limitations (#638)
elastic/stack-docs: docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

[float]
[[dfa-update-limitations]]

> === A {dfanalytics-job} configuration cannot be updated

Reviewer: Is there some part of the job that *can* be updated? If not, I suggest simplifying to something like this:
```suggestion
=== {dfanalytics-jobs-cap} cannot be updated
```

dedicated for {ml} processes. For general {ml} settings, see 
{ref}/ml-settings.html[{ml-cap} settings in {es}].

[float]
[[dfa-missing-fields-limitations]]

> === Documents with missing values in analyzed fields will be skipped

Reviewer: ```suggestion
=== Documents with missing values in analyzed fields are skipped
```
